### DISCUSSION TITLE
Cloning and Setting Up Webpack: Almost Amazon

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#135] [Help Me!] [answered]
Dani Crosby

Have you searched the other help tickets? Yes

Describe your issue: I noticed a few of my teammates last night had a cloned version of the Almost Amazon where there were 10 issues already doled out, mine did not. I had all of my code working yesterday and understand the process, so I decided to try and grab this other template with the issues. I deleted my previous work (more practice the better right!) and started from scratch, even in Firebase. Now when I try to run NPM i, I get errors and upoloads a locked version of json and won'lt let me npm start as well. 

I have searched what these errors mean and install can't find a solution. I have tried a few of the terminal commands in the following threads with no luck. NPM start will not work either and when I straight up SERVE, I'm just seeing the directory...

npm ERR! enoent This is related to npm not being able to find a file.
enoent ENOENT: no such file or directory, open '/Users/mrsdanicrosby/workspace/foundations/inclass/almost-amazon/package.json'

https://stackoverflow.com/questions/45531633/npm-warn-enoent-enoent-no-such-file-or-directory-open-c-users-nuwanst-packag

https://stackoverflow.com/questions/20753550/enoent-no-such-file-or-directory

https://stackoverflow.com/questions/46261063/npm-start-doesnt-work

https://stackoverflow.com/questions/54124033/deleting-package-lock-json-to-resolve-conflicts-quickly/54127283

**Comment by drteresavasquez** [comment]
They did not clone. It was a card to be moved over inside of CodeTracker that makes the repo and tickets. Did you move over your card? If so I can try and see if I can find it to delete so you can start over. For future, do not delete repos from CodeTracker to start over unless we know first so we can get things ready on our end.

**Reply by danicrosby** [reply]
Please resend if you don't mind! i'll try that first and touch base if that doesn't work. Thank you!

**Reply by danicrosby** [reply]
Errrrppp... Looks like I moved the card to 'done' which I'm clearly not...

**Reply by drteresavasquez** [reply]
No worries. I'm not at a computer so it will be about an hour before I can do that but once we get it set up all you shouldn't have to do is copy over your keys to the env file since you already did the Firebase. Firebase only allows you a certain number of projects unless you contact them so don't make anymore.

**Reply by danicrosby** [reply]
Oh shoot! Ok! I'll stand down.... LOL

**Comment by danicrosby** [comment, marked as answer]
OK!!! SO i figured this out! 

1. I noticed I was trying to run an NPM i on an almost-amazon folder that didn't register a main/master. So typed 'git init' on the command line. I'm guessing I wasn't able to install the NPM i because I wasn't on a legit branch.

2. Once I changed into a branch, I tried to save my changes, all kinds of angry messages from the terminal again. This time stating that I already had a almost-amazon, and sure enough- the amazon folder I was making changes to was actually nested INSIDE a previous amazon folder-- SOOOO

3. I moved all the relevant files in to the outer amazon file and deleted the inner one including the locked version on json.

4. I was able to successfully run the NPM i and also run NPM Start with no issue... WHEW!

**Reply by drteresavasquez** [reply]
So you're good?

**Reply by drteresavasquez** [reply]
@danicrosby your card should be back to the backlog. Let me know if you have issues moving it over.
